src/shared/lib/FisherExactTestCalculator.ts

```diff
@@ -51,3 +51,49 @@ export function getCumulativePValue(
     }
     return p;
 }
+
+export function getTwoTailedPValue(
```

Comment:
we definately need to test the crap out of this. did you make sure we didn't already have an implementation of this somewhere?

Reply:
Also, if we have to keep this function can we replace for loops with forEach, map, etc wherever applicable?